back/engines/commercial/idea_custom_fields/app/policies/idea_custom_fields/idea_custom_field_policy.rb

```diff
@@ -14,6 +14,10 @@ def update_all?
       can_configure_custom_fields? record
     end
 
+    def as_geojson?
+      update_all?
```

Comment:
I'd probably just use the same as the rest of the policy for consistency:
```suggestion
      can_configure_custom_fields? record
```